src/css/main.css

```diff
@@ -22,3 +22,13 @@
 .clip-bottom-8 {
   clip-path: polygon(0 0, 100% 0, 100% calc(100% - 8px), 0 calc(100% - 8px));
 }
+
+.unorder-list li {
+  padding-left: 16px;
+}
+
+.unorder-list li::before {
+  content: "•";
+  padding-right: 8px;
+  color: currentColor;
+}
```

Comment:
Maybe instead of these "custom bullet points", we could use Tailwind's `list-inside` and `list-disc` in the HTML on the `<ul>`? The dots are a bit bigger, but I think it looks okay?

Current:
![image](https://user-images.githubusercontent.com/462459/172897702-5ff58871-4013-4f89-a588-22c0c379aa96.png)

Suggested (also with `pl-2` for extra padding on the left):
![image](https://user-images.githubusercontent.com/462459/172897736-b373bfb4-8a37-4f5c-869d-156001ae4e56.png)